elasticdl/python/worker/worker.py

```diff
@@ -181,9 +188,10 @@ def get_model(self, version, method):
 
         # Assumes all trainable variables exist in model.param.
         for tensor_pb in model.param:
-            self._non_embed_vars[tensor_pb.name].assign(
-                tensor_to_ndarray(tensor_pb)
-            )
+            if tensor_pb.name in self._non_embed_vars:
+                self._non_embed_vars[tensor_pb.name].assign(
+                    tensor_to_ndarray(tensor_pb)
+                )
```

Comment:
This line has a conflict with the above comment. And just ignore the variables that are not in `worker._non_embed_vars` without doing anything seems a little dangerous.

In fact, it seems that this has already be fixed in #1380 (this PR is merged after you submit your PR). So maybe remove this `if`?